src/query/sql/src/executor/format.rs

```diff
@@ -204,6 +210,36 @@ fn table_scan_to_format_tree(
         }
     };
     children.push(FormatTreeNode::new(push_downs));
+    // Aggregating index
+    if let Some(agg_index) = agg_index {
+        let metadata = metadata.read();
+        let (_, agg_index_sql, _) = metadata
+            .get_agg_indexes(&table_name)
+            .unwrap()
+            .iter()
+            .find(|(index, _, _)| *index == agg_index.index_id)
+            .unwrap();
+
+        children.push(FormatTreeNode::new(format!(
+            "aggregating index: [{agg_index_sql}]"
+        )));
+
+        let agg_sel = agg_index
+            .selection
+            .iter()
+            .map(|expr| expr.as_expr(&BUILTIN_FUNCTIONS).sql_display())
+            .join(", ");
+        let agg_filter = agg_index
+            .filter
+            .as_ref()
+            .map(|f| f.as_expr(&BUILTIN_FUNCTIONS).sql_display());
```

Comment:
filters also need join ", " here?

Reply:
They are already combined into one expression by "and".